src/test/java/com/wikia/webdriver/common/core/helpers/UserWithEmailPool.java

```diff
@@ -3,20 +3,27 @@
 
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.properties.Credentials;
-import lombok.Getter;
+import lombok.Value;
 
-public class ForgottenPasswordUserPool {
+@Value
+public class UserWithEmailPool {
 
   private static final Credentials CREDENTIALS = Configuration.getCredentials();
 
-  @Getter
   private UserWithEmail user = new UserWithEmail(User.FORGOTTEN_PASSWORD,
     CREDENTIALS.forgottenPasswordEmail1Address,
     CREDENTIALS.forgottenPasswordEmail1Password);
 
-  @Getter
   private UserWithEmail userWithSpaces = new UserWithEmail(User.FORGOTTEN_PASSWORD_SPACES,
     CREDENTIALS.forgottenPasswordEmail2Address,
     CREDENTIALS.forgottenPasswordEmail2Password);
 
+  private UserWithEmail emailOnlyUser1 = new UserWithEmail(null,
+    CREDENTIALS.emailQaart1,
+    CREDENTIALS.emailPasswordQaart1);
+
+  private UserWithEmail emailOnlyUser2 = new UserWithEmail(null,
+    CREDENTIALS.emailQaart2,
+    CREDENTIALS.emailPasswordQaart2);
```

Comment:
I think it might be a bit more appropriate to make this a factory (unless there's something I don't know):
```
public class UserWithEmailFactory {
    public getUserWithSpaces() { ...
...
```